### PR TITLE
oomd fixups

### PR DESCRIPTION
**Pull request by anitazha**
Applied most of the suggestions from @poettering's post-merge review of #15206. The only thing missing besides the `man/org.freedesktop.systemd1.xml` property documentation (which I'm not sure are necessary) is the rename from `ManagedOOMMemoryPressureLimitPercent` -> `ManagedOOMMemoryPressureScale`

Also sprinkled in a couple of my own oomd-related fixes.

### CODE REVIEW

File: src/oom/test-oomd-util.c [hunk 1]
```diff
@@ -52,19 +53,20 @@ static void test_oomd_cgroup_kill(void) {
          * by the test so that pid1 doesn't delete it before we can read the xattrs. */
         cgroup = path_join(cgroup_root, "oomdkilltest");
         assert(cgroup);
+        assert_se(cg_create(SYSTEMD_CGROUP_CONTROLLER, cgroup) >= 0);
 
         /* If we don't have permissions to set xattrs we're likely in a userns or missing capabilities */
-        if (cg_set_xattr(SYSTEMD_CGROUP_CONTROLLER, cgroup, "user.oomd_test", "test", 4, 0) == -EPERM)
-                return (void) log_tests_skipped("no permissions to set user xattrs");
+        r = cg_set_xattr(SYSTEMD_CGROUP_CONTROLLER, cgroup, "user.oomd_test", "test", 4, 0);
+        if (IN_SET(r, -EPERM, -ENOTSUP))
```
Comment (keszybz) [inline src/oom/test-oomd-util.c:60]: Maybe `ERRNO_IS_NOT_SUPPORTED(r)` ?

**Review by keszybz — COMMENTED** [review]
Looks OK.

File: src/oom/test-oomd-util.c (hunk 1 above)
Comment (poettering) [reply src/oom/test-oomd-util.c:60]: Yes, please, let's use ERRNO_IS_NOT_SUPPORTED(r) in new code, please

**Comment by poettering** [comment]
lgtm, too!

**Pull request merged**